crowbar_framework/app/models/node_object.rb

```diff
@@ -1451,6 +1451,26 @@ def disk_release!(device, owner)
     disk_release(device, owner) and save
   end
 
+  def check_unclaimed_disks
+    unclamed_disks = Array.new
+    crowbar_wall[:claimed_disks].each do |disk, _claim|
+      path, device_name = disk.split("/")[-2..-1]
+
+      unless disk =~ /^\/dev\/disk\//
+        unclamed_disks.push(device_name) unless node[:block_device][path].nil?
+        next
+      end
+
+      devices = node[:block_device].map do |device, _claim|
```

Comment:
Yes, will need to rename the _claim variable to something that makes more sense.
I can try to do it that way but IIRC there was some problem with that. But I can check again.